_notebooks/2020-03-27-covid19-progress-projections.ipynb

```diff
@@ -0,0 +1,2126 @@
+{
```

Comment:
<p> </p><p> </p><ul><li><code>ICU capacities differ by country but seem to be in the range of&nbsp;<a href="https://www.forbes.com/sites/niallmccarthy/2020/03/12/the-countries-with-the-most-critical-care-beds-per-capita-infographic/" target="_blank">2-10 for developed countries</a>.</code> I linked to a wikipedia article where you can find this data in another comment and add that into this dashboard, so you can remove this bullet.</li><li>This definition is ambiguous - how is this ratio calculated? Can we provide a more precise calculation?  <code><strong>Estimated testing bias ratio from deaths</strong>: estimated ratio of how reduced is the testing coverage relative to countries which performed comprehensive testing.  </code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/github/covid19-dashboard/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>